sdk/python/feast/type_map.py

```diff
@@ -774,6 +776,10 @@ def spark_to_feast_value_type(spark_type_as_str: str) -> ValueType:
         "array<boolean>": ValueType.BOOL_LIST,
         "array<timestamp>": ValueType.UNIX_TIMESTAMP_LIST,
     }
+    decimal_regex_pattern = r"^decimal\([0-9]{1,2},[0-9]{1,2}\)$"
```

Comment:
startswith("decimal") is probably more appropriate for now if we don't care about the precision.